mkdocs.yml

```diff
@@ -13,6 +13,8 @@ pages:
   - 'Build': 'mechanisms/build.md'
   - 'Deployment': 'mechanisms/deployment.md'
 - Plugins: plugins.md
+- Bundled plugins:
```

Comment:
On line 17 the string with a space uses single quotes, but this one does not. Is it required?